vmas/simulator/environment/environment.py

```diff
@@ -770,6 +773,53 @@ def render(
         # render to display or array
         return self.viewer.render(return_rgb_array=mode == "rgb_array")
 
+    def plot_boundary(self):
+        # Include boundaries in the rendering
+        from vmas.simulator import rendering
+        from vmas.simulator.rendering import Line
+        from vmas.simulator.utils import Color
+
+        # Check boundary limits
+        if self.world.x_semidim is not None or self.world.y_semidim is not None:
+
+            infinite_value = 1e7
```

Comment:
```suggestion
            infinite_value = 100
```

for some reason 10e7 does not work for me (no plotting)